mep/common/templates/common/widgets/checkbox_fieldset.html

```diff
@@ -0,0 +1,18 @@
+{% with id=widget.attrs.id name=widget.attrs.name %}
+    <fieldset class="facet">
+    <legend>{{ name|title }}</legend>
+    {% for group, options, index in widget.optgroups %}
+        {% for option in options %}
+           <input type="checkbox"
```

Comment:
Can we put these on the same line? This structure makes the template more readable but it's not great for the generated html. I would suggest using `spaceless` but, I'm pretty sure that doesn't help any here.

Reply:
Sure. I'm happy to collapse them. I erred on the side of readablility for our use, since a gzipped HTML file won't suffer for extra space.

Reply:
It's not for space but for readability in the html output, which could matter when we're debugging issues and dealing with a large number of facets. (I remember having to do some of this with the derrida code, hopefully we won't have to here!)